roles/nexus-ocp/defaults/main.yml

```diff
@@ -22,3 +22,5 @@ _nexus_expose_uri: ""
 _nexus_oauth: True
 _nexus_route: "{{ _nexus_expose_uri }}"
 _nexus_ssl: True
+
+_nexus_jvm_memory: 512m
```

Comment:
One more- could you add a newline to the end here? Otherwise linting will fail during release

Reply:
Done :smile: